rotkehlchen/chain/ethereum/decoding/decoder.py

```diff
@@ -431,10 +431,11 @@ def _maybe_decode_simple_transactions(
             ))
             return events
 
-        if amount == ZERO:
-            return events
-
         preposition = 'to' if verb == 'Send' else 'from'
+        if amount == ZERO:
+            notes = 'There was no ETH transfer in this transaction'
```

Comment:
This message seems confusing at least to me.

```
The amount of ETH transfered in this transaction was 0.
```

From the curren note I understand that all transactions should have an eth transfer and this one doesn't have it

Reply:
Yeah, makes sense. Do you have any ideas for this note?

Reply:
The one I wrote is the one that I believe is the clearest. Maybe

```
This transaction included a transfer of ETH with value 0
```

Reply:
Ah, sorry, thought you just pasted my version in the comment. But hm... there is no eth transfer in such transactions, so sounds more confusing to me than the original version

Reply:
So maybe "This transactions didn't contain any trackable event"

Reply:
I guess would be a great option if it was the only event in transaction. But the problem is that for consistency, info about amount of eth "transfered" will be present in all transactions

Reply:
A 0 value transfer is fine. Transferred 0 ETH to XXX.